analyzer/src/funTest/assets/projects/synthetic/gradle-all-dependencies-expected-result.yml

```diff
@@ -2,13 +2,13 @@
 repository:
   vcs:
     type: "Git"
-    url: "<REPLACE_URL>"
-    revision: "<REPLACE_REVISION>"
+    url: "https://github.com/oss-review-toolkit/ort.git"
```

Comment:
I'm not sure if this file was ever used by an analyzer test, but as it is also not used in the CLI tests anymore since f63e3f9 it would also make sense to move it to the `model` module.

Reply:
I'll do this in a follow up PR.